### PR TITLE
Support autologin after verification

### PR DESCRIPTION
**Pull request by gugu**
And also support django 1.10

### CODE REVIEW

File: email_confirm_la/urls.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # coding: utf-8
 
-from django.conf.urls import patterns, url
+from django.conf.urls import url
+from .views import confirm_email
```
Comment (vinta) [inline email_confirm_la/urls.py]: I prefer absolute imports at least for consistency.

File: email_confirm_la/conf.py [hunk 2]
```diff
@@ -10,6 +10,7 @@ class DefaultConfigs(object):
     EMAIL_CONFIRM_LA_CONFIRM_EXPIRE_SEC = 60 * 60 * 24 * 1  # 1 day
     EMAIL_CONFIRM_LA_CONFIRM_URL_REVERSE_NAME = 'email_confirm_la:confirm_email'
     EMAIL_CONFIRM_LA_TEMPLATE_CONTEXT = {}
+    EMAIL_CONFIRM_LA_AUTOLOGIN = False
```
Comment (vinta) [inline email_confirm_la/conf.py]: I think we could set default to `True`

File: email_confirm_la/views.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # coding: utf-8
 
 from django.shortcuts import render
+from django.contrib.auth import login
```
Comment (vinta) [inline email_confirm_la/views.py]: It would be good to sort imports in alphabetical order.

File: email_confirm_la/views.py [hunk 4]
```diff
@@ -21,4 +22,8 @@ def confirm_email(request, confirmation_key):
     except EmailConfirmation.ExpiredError:
         return render(request, 'email_confirm_la/email_confirmation_expiration.html', context)
 
+    if configs.EMAIL_CONFIRM_LA_AUTOLOGIN:
+        email_confirmation.content_object.backend = 'django.contrib.auth.backends.ModelBackend'
+        login(request, email_confirmation.content_object)
```
Comment (vinta) [inline email_confirm_la/views.py]: You should check whether if `email_confirmation.content_object`is a `User` object, after all `content_object` is a `GenericForeignKey`.

Comment (vinta) [inline email_confirm_la/views.py]: or `try except` the `login()`.

**Comment by vinta** [comment]
Thank you for sending this helpful pull request!

File: email_confirm_la/conf.py (hunk 2 above)
Comment (vinta) [inline email_confirm_la/conf.py]: Oh no, I take back my words, it should be `False`.

**Comment by gugu** [comment]
@vinta fixed

**Pull request merged**